setup.py

```diff
@@ -5,30 +5,25 @@
 from setuptools import setup
 import versioneer
 
-# NOTE: These are tested in `continuous_integration/travis/test_imports.sh` If
+# NOTE: These are tested in `continuous_integration/test_imports.sh` If
 # you modify these, make sure to change the corresponding line there.
 extras_require = {
-    "array": ["numpy >= 1.15.1", "toolz >= 0.8.2"],
-    "bag": [
-        "cloudpickle >= 0.2.2",
-        "fsspec >= 0.6.0",
-        "toolz >= 0.8.2",
-        "partd >= 0.3.10",
-    ],
-    "dataframe": [
-        "numpy >= 1.15.1",
-        "pandas >= 0.25.0",
-        "toolz >= 0.8.2",
-        "partd >= 0.3.10",
-        "fsspec >= 0.6.0",
-    ],
+    "array": ["numpy >= 1.15.1"],
+    "bag": [],  # keeping for backwards compatibility
+    "dataframe": ["numpy >= 1.15.1", "pandas >= 0.25.0"],
     "distributed": ["distributed >= 2021.03.0"],
     "diagnostics": ["bokeh >= 1.0.0, != 2.0.0"],
-    "delayed": ["cloudpickle >= 0.2.2", "toolz >= 0.8.2"],
+    "delayed": [],  # keeping for backwards compatibility
 }
 extras_require["complete"] = sorted({v for req in extras_require.values() for v in req})
 
-install_requires = ["pyyaml"]
+install_requires = [
+    "pyyaml",
+    "cloudpickle >= 1.1.1",
```

Comment:
I wonder what we think about bumping this to 1.5.0. That version added support for the `pickle5` backport package. Admittedly that is only needed for Python pre-3.8, which is just Python 3.7 for Dask now

Reply:
I don't think dask/dask has any need to directly acknowledge pickle protocol 5? 1.5.0 is very recent...

Reply:
distributed pins to 1.5.0, but since that is less than a year old it seemed like overkill for dask/dask.

Reply:
Cloudpickle will use it under-the-hood if installed

Yeah I don't have strong feelings here. Just wanted to raise for discussion. This also will become irrelevant once Python 3.7 is dropped (guessing that will coincide with the Python 3.10 release)